fpdcc/ccfp-asset-dashboard
# Improved table filtering + add csv export button for projects by district filtering

## Overview
Closes:
- #145 
- #154
- #155

## Testing Instructions
- visit https://ccfp-asset-d-feature-im-llkt58.herokuapp.com
- credentials: `ccfp-asset-dashboard test user`
- **test projects filtering**:
  - visit root page
  - do some filtering on the projects
  - click "reset" button and make sure the table resets
  - do more filtering
  - soft refresh page and make sure the table resets
- **test projects by district filtering**:
  - visit https://ccfp-asset-d-feature-im-llkt58.herokuapp.com/projects/districts/
  - make sure the OR filtering works: filter by one district that exists in the data and one that doesn't
  - make sure the reset button works
  - test the csv export works
